test/e2e-test-application/cypress/e2e/tests/1-angular/navigation-4.cy.js

```diff
@@ -290,4 +290,33 @@ describe('Navigation', () => {
       );
     });
   });
+
+  describe('Split View and Modal', () => {
+    beforeEach(() => {
+      cy.visitLoggedIn('/projects/pr2');
+    });
+
+    it('keeps a split view open if a modal with the same view is opened and closed again', () => {
+      cy.expectPathToBe('/projects/pr2');
+      cy.getIframeBody().then(result => {
+        cy.wrap(result)
+          .find('a[data-testid="open-overview-in-splitview"]')
+          .click();
+
+        cy.wrap(result)
+          .find('a[data-testid="open-overview-in-modal"]')
+          .click();
+      });
+
+      // Modal is open
+      cy.get('div[data-testid="modal-mf"]').should('exist');
+
+      // Close modal
+      cy.get('button[data-testid="lui-modal-index-0"]').click();
+
+      cy.get('#splitViewContainer').should('exist');
+      cy.get('#splitViewDragger').should('exist');
+      cy.get('div[data-testid="modal-mf"]').should('not.exist');
```

Comment:
This works, but it could generate flaky tests. 

It would be better if you put this section inside of the **then** callback above to avoid a race condition when should('exist') is called before the modal/splitview is open

Reply:
Good catch, thanks!